lib/http2.c

```diff
@@ -162,6 +162,7 @@ static CURLcode http2_disconnect(struct connectdata *conn,
   nghttp2_session_del(c->h2);
   Curl_safefree(c->inbuf);
   http2_stream_free(conn->data->req.protop);
+  conn->data->state.drain = 0;
```

Comment:
drain_total has to track drain but I guess at this point it no longer matters?

Reply:
Exactly, the global drain counter is taken care of already. This clearing is to make sure that when this transfer is put back on a fresh/another connection, it starts from zero again.